Formula/babelfish.rb

```diff
@@ -27,6 +27,22 @@ def install
     fish_function.install "babel.fish"
   end
 
+  def caveats
+    <<~CAVEATS
+      The shell hook has been installed to which you can use by running:
+
+        $ source /opt/homebrew/share/fish/vendor_functions.d/babel.fish
+
+      This allows you to run a bash script like this:
+
+        $ source chruby.sh
+        $ chruby
+           ruby-2.5
+           ruby-2.6
+           ruby-2.7
+       CAVEATS
```

Comment:
Fix `brew style`'s alignment:
```suggestion
    CAVEATS
```

Reply:
this was already fixed in 255e9f810bb9a56487926edc89963889e9d91d10